pkg/crypto/certificatemanager/certificate_manager.go

```diff
@@ -87,9 +87,7 @@ func (m *Manager) GetTLSContext(ctx context.Context, tlsCtx *api.TLSContext, ns
 		// Give priority to local certificates
 		certPath := filepath.Join(m.rootPath, ns, name)
 		files, ioErr := ioutil.ReadDir(certPath)
-		if ioErr != nil {
-			err = fmt.Errorf("Certificates directory %s not found (%s)", certPath, ioErr)
-		} else {
+		if ioErr == nil {
```

Comment:
How do we know if the directories didn't exist in the host in the first place?

Reply:
The `err` set here was overwritten below, so in the current version we did not know it either. Also, the directory is not required to exist. Maybe we should require the existence of the directory if it is configured on the command line?

Reply:
> The `err` set here was overwritten below, so in the current version we did not know it either. Also, the directory is not required to exist. Maybe we should require the existence of the directory if it is configured on the command line?

By "command line" you mean in the CLI or the k8s policies descriptors?

Reply:
I mean a cilium-agent command parameter (like `--debug`).